### ISSUE TITLE
Pip takes a huge amount of time to install tensorflow-quantum

### ISSUE THREAD

**Issue by sleepingcat4** [tensorflow/quantum#700]
I've been trying to install Tensorflow Quantum as outlined by Tensorflow documentation, here: [https://www.tensorflow.org/quantum/tutorials/hello_many_worlds]

Unfortunately, pip states it's trying to find the required version and it's been 6 hours since I executed the code and still not fully installed. I even changed my runtime to TPU. Things were bit good but same story happened again. I even specified Tensorflow Quantum version "pip install tensorflow-quantum==0.6.1" while installing using 'pip' command in another attempt. Unfortunately no luck! Can anyone explain how this issue can be resolved?

**Comment by lockwo** [comment]
What version of pip are you using? I just tested with `pip 22.2` and it installed about as quickly as expected (<1 min). https://github.com/tensorflow/quantum/issues/696 seems to indicate this might be an issue with pip.

**Comment by sleepingcat4** [comment]
At first I used the build-In version of pip that was pre-installed within Google Collab but later I had updated it after I received an error. That said, something didn't matched with the existing component's version and as a result, pip was unable to resolve it. But after seeing installed libraries list, I saw it Tensorflow-quantum was installed. But, to solve that version issue pip showed I needed to have root user permission.

**Comment by lockwo** [comment]
Yes, I also saw that conflict, but it doesn't seem to have an effect on execution. I just ran the intro tutorial colab with no problems (even with the conflict). That being said, this is probably something worth looking into for the team.

**Comment by lockwo** [comment]
Any updates on this or should it be closed?

**Comment by sleepingcat4** [comment]
I don't think Google Fixed it yet! Since there's conflict even if you install using the latest version of "pip". I think we should let have this open so that Google can identify the problem and solve it in the next version : )

**Comment by mhucka** [comment]
Version 0.7.5 of TensorFlow Quantum [has been released](https://github.com/tensorflow/quantum/releases/tag/v0.7.5) and is now available via pip/PyPI.org. Please make sure to read the [release notes](https://github.com/tensorflow/quantum/releases/tag/v0.7.5) for the installation instructions and other notes.

This new version also works in Google Colab; however, it requires a couple of steps to work around the fact that Colab comes with different versions of TensorFlow and NumPy preinstalled. Please [refer to issue #959](https://github.com/tensorflow/quantum/issues/959) for more details and guidance regarding Colab.

I am going to close this and other similar issues today. If you encounter problems with installing or running TensorFlow Quantum 0.7.5 on Linux x86_64 systems, please open a new issue; doing so will make it easier to track new problems.
